pages/about.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="/styles/style.css" rel="stylesheet">
 </head>
-<body>
-  
-</body>
+  <body>
+    <header>
+      <h1>About me</h1>
+      <nav>
+        <ul>
+          <li>
+            <a href="index.html">Home</a>
+          </li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <a href="portfolio.html">Portfolio</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="flex-grow-1">
+      <section class="container-about">
+      
+        <div>
```

Comment:
This div tag doesn't have id or class on it. Are you using it style the page? It doesn't seem like it. 

If you don't use it for styling it can be removed since it's not semantic